Explorer/Assets/DCL/AvatarRendering/AvatarShape/Rendering/TextureArray/TextureArrayContainerFactory.cs

```diff
@@ -69,12 +70,16 @@ public TextureArrayContainer Create(Shader shader, IReadOnlyList<int> defaultRes
             };
         }
 
-        public TextureArrayContainer Create(string shaderName, IReadOnlyList<int> defaultResolutions, TextureFormat format, int minArraySize)
+        public TextureArrayContainer Create(string shaderName, IReadOnlyList<TextureArrayResolutionDescriptor> defaultResolutions, TextureFormat format,
```

Comment:
Let's change its name to `CreateSceneLOD`, is it for LODs only, right?

Reply:
Sounds right